packages/ethereum-contracts/contracts/interfaces/agreements/gdav1/ISuperfluidPool.sol

```diff
@@ -83,9 +65,14 @@ interface ISuperfluidPool is IERC20 {
     /// @param time The time to query
     function getDisconnectedBalance(uint32 time) external view returns (int256 balance);
 
+    /// @notice The total amount received by `memberAddr` in the pool
```

Comment:
Should make it more clear whether this includes both the connected and unconnected parts